app/assets/javascripts/components/story/DatePicker.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+
+export default class StoryDatePicker extends React.Component {
```

Comment:
What do you think of rename the file so it matches your class name? I think it would make it easier to find.